src/pages/wallet/VestingScheduleTable.tsx

```diff
@@ -24,7 +28,7 @@ const VestingScheduleTable = ({ type, schedule }: ParsedVestingSchedule) => {
         {
           title: "Amount",
           dataIndex: "amount",
-          render: (value) => <Read amount={value} />,
+          render: (value) => <Read amount={value} denom={denom} />,
```

Comment:
Token symbol shouldn't display in VestingScheduleTable:
<img width="403" alt="Screenshot 2023-08-18 at 10 24 07 AM" src="https://github.com/terra-money/station/assets/30275692/a497681e-484e-46c0-8e4c-f76f59b5d61c">

Production:

<img width="406" alt="Screenshot 2023-08-18 at 10 24 54 AM" src="https://github.com/terra-money/station/assets/30275692/14f7cb25-4ac8-49c8-8a18-93a5f22a764d">

Reply:
fixed